AppTests/App/Logic/AppSetupLogicTests.swift

```diff
@@ -39,20 +39,7 @@ final class AppSetupLogicTests: XCTestCase {
     var state = AppState()
 
     // swiftlint:disable force_unwrapping
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: AppTests/App/Logic/AppSetupLogicTests.swift;
  Line: 41;
-->

- :warning: swiftlint (0.39.2): SwiftLint rule 'force_unwrapping' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)